geometry_msgs/PoseStamped pose
string eef
---
